### PR TITLE
Add sample for amp-carousel tracking

### PR DESCRIPTION
**Pull request by sebastianbenz**
@kul3r4 PTAL
@rudygalfi @avimehta @eike-hass FYI

### CODE REVIEW

File: src/20_Components/amp-analytics.html [hunk 1]
```diff
@@ -35,6 +37,14 @@
...
+    amp-carousel .slide {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      font-size: 20px;
+      background: green;
+      color: white;
+    }
     </style>
   <script async src="https://cdn.ampproject.org/v0.js"></script>
```
Comment (kul3r4) [inline src/20_Components/amp-analytics.html]: should we move this on top like the other samples?

File: src/20_Components/amp-analytics.html [hunk 2]
```diff
@@ -221,5 +231,79 @@
...
+        "vars": {
+          "eventId": "carousel-change-${toSlide}"
+        }
+      }
+    }
+  }
+  </script>
+  </amp-analytics>
+  <!--
+The `amp-carousel-next` event is issued when there is a travesal to the next slide: -->
```
Comment (kul3r4) [inline src/20_Components/amp-analytics.html]: travesal? is it trasversal

File: src/20_Components/amp-analytics.html (hunk 1 above)
Comment (sebastianbenz) [inline src/20_Components/amp-analytics.html]: That was part of the other PR

File: src/20_Components/amp-analytics.html (hunk 2 above)
Comment (sebastianbenz) [inline src/20_Components/amp-analytics.html]: Done. It's actually traversal

File: src/20_Components/amp-analytics.html [hunk 3]
```diff
@@ -221,5 +231,79 @@
...
+    <div class="slide" data-slide-id="slide1">Slide 1</div>
+    <div class="slide" data-slide-id="slide2">Slide 2</div>
+    <div class="slide">Slide 3</div>
+  </amp-carousel>
+
+  <!--
+You can use the `fromSlide` and `toSlide` [variables](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md) for tracking which slides are viewed. The value is either taken from the `data-slide-id` attribute of the slide when present, else it represents the index of the slide (starting from 0).
+  -->
+  <!--
+The `amp-carousel-change` event is issued when there is any change in the slide that is curently visible:
```
Comment (sebastianbenz) [inline src/20_Components/amp-analytics.html]: done

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

**Pull request merged**